src/lib/Menu/ContentRightSidebarBuilder.php

```diff
@@ -443,6 +554,63 @@ private function getCopySubtreeLimit(): int
             'subtree_operations.copy_subtree.limit'
         );
     }
+
+    /**
+     * @param \Ibexa\Contracts\Core\Repository\Values\Content\Location|null $location
+     * @param \Ibexa\Contracts\Core\Repository\Values\Content\Content $content
+     * @param \Ibexa\Contracts\Core\Repository\Values\Content\Language $language
+     * @param array $options
+     *
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\BadStateException
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\InvalidArgumentException
+     */
+    private function getContentPreviewItem(
+        ?Location $location,
+        Content $content,
+        Language $language,
+        array $options,
+        string $idPostfix = ''
+    ): ItemInterface {
+        if ($location === null) {
+            throw new \InvalidArgumentException('location cannot be null');
```

Comment:
Why typehinting to nullable type then?

Reply:
yeah, but there it works with nullable location, here you say you can pass null just to throw exception one line after that. You can achieve the same with changing `?Location` to just `Location`.